public/scripts/form.js

```diff
@@ -29,14 +30,20 @@ $(function () {
 
   socket.on('logs', function (data) {
     $('#messages').append($('<li class="info">').text(data.data));
-    $("#progress").css("width", data.donePercent + "%");
+    if (data.donePercent >= 100) {
+      $("#progress").css("width", "90%");
```

Comment:
doesn't this mean that if progress goes from 95 to 100, the bar would actually roll back to 90?